### PR TITLE
New UI at bottom of page

### PR DESCRIPTION
**Pull request by benfrancis314**

### CODE REVIEW

**Comment by benfrancis314** [comment]
NOTE: This is branched from the tutorial push

**Comment by benfrancis314** [comment]
Also added blue "gentle" highlighter option. Easy to implement and take away if we don't like it

File: extension/content_script.js [hunk 1]
```diff
@@ -470,6 +470,9 @@ function toggleKeywordSettings() {
 		settingsView.changeSetting('keyword', 'bright'); 
 	}
 	else if (currentKeywordStyle == 'bright') { 
+		settingsView.changeSetting('keyword', 'gentle');
```
Comment (sjonany) [inline extension/content_script.js]: dont have to do this but fyi a nice one-liner

keywordStyleOrders = [off, light, bright, gentle]
nextStyle = keywordStyleOrders[(keywordStyleOrders.indexOf(currentStyle) + 1) % keywordStyleOrders.length]

**Review by sjonany — APPROVED** [review]
LGTM! 
Will try later in master. Thank you for doing this!

File: extension/content_script.js (hunk 1 above)
Comment (benfrancis314) [reply extension/content_script.js]: V cool idea, thanks!

**Pull request merged**